components/raftstore/src/store/peer.rs

```diff
@@ -1536,7 +1547,7 @@ where
 
     #[inline]
     pub fn is_initialized(&self) -> bool {
-        self.get_store().is_initialized()
+        self.get_store().is_initialized() && self.initialized
```

Comment:
This is confusing. Hard to know whether use the wrapped version or peer storage version. Better change the name or add a new method.